src/components/tab-bar/tab-bar.tsx

```diff
@@ -143,6 +143,7 @@ export class TabBar {
                 <span class="mdc-tab__content">
                     {this.renderIcon(tab)}
                     <span class="mdc-tab__text-label">{tab.text}</span>
+                    <limel-badge label={tab.badge} />
```

Comment:
This needs to be conditional. If there is no badge for the tab, this should not be rendered